packages/liveblocks-client/src/LiveList.test.ts

```diff
@@ -108,38 +138,10 @@ describe("LiveList", () => {
       });
 
       items.push(new LiveObject({ a: 0 }));
-      assert(
```

Comment:
I removed updates assertion in this unit tests because it already is doing a lot. Instead I created a `prepareStorageUpdateTest` function that help us test only tests updates.